x/gamm/pool-models/stableswap/amm_test.go

```diff
@@ -981,3 +981,65 @@ func TestJoinPoolSharesInternal(t *testing.T) {
 		})
 	}
 }
+
+func TestTargetKCalculator(t *testing.T) {
```

Comment:
I don;t understand what this test is actually testing

Reply:
As I understand, maybe @p0mvn want to check if given the same conditions. Will the results of calculating be consistent ? as he have said in the issue that I want to close.